F/FastJet/build_tarballs.jl

```diff
@@ -31,6 +31,16 @@ fi
 ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-auto-ptr
 make -j ${nprocs} "${FLAGS[@]}"
 make install
+tail -n 340 COPYING > LICENSE
+install_license LICENSE
+
+cd $WORKSPACE/srcdir/fjcontrib-*/
+for name in ClusteringVetoPlugin ConstituentSubtractor EnergyCorrelator FlavorCone GenericSubtractor JetCleanser JetFFMoments JetsWithoutJets LundPlane Nsubjettiness QCDAwarePlugin RecursiveTools ScJet SoftKiller SubjetCounting ValenciaPlugin VariableR
+do
+    cd $name
+    rm example*
+    c++ -fPIC -shared -I${includedir} -O3 -Wall ${LDFLAGS} [A-Z]*.cc -o "${libdir}/lib${name}.${dlext}"
```

Comment:
Sorry, I removed too much:
```suggestion
    c++ -fPIC -shared -I${includedir} -O3 -Wall ${LDFLAGS} [A-Z]*.cc -o "${libdir}/lib${name}.${dlext}"
    cd ..
```
:grimacing: